tests/test_kotlin.py

```diff
@@ -174,3 +174,55 @@ def test_glean_namespace(tmpdir):
         assert content.count(
             "import Bar.private.CounterMetricType"
         ) == 1
+
+
+def test_gecko_datapoints(tmpdir):
+    """Test translating metrics to Kotlin files."""
+    tmpdir = Path(tmpdir)
+
+    translate.translate(
+        ROOT / "data" / "gecko.yaml",
+        'kotlin',
+        tmpdir,
+        {'glean_namespace': 'Bar'},
+        {'allow_reserved': True}
+    )
+
+    assert (
+        set(x.name for x in tmpdir.iterdir()) ==
+        set([
+            'GeckoImports.kt',
+            'NonGeckoMetrics.kt'
+        ])
+    )
+
+    # Make sure descriptions made it in
+    with open(tmpdir / 'GeckoImports.kt', 'r', encoding='utf-8') as fd:
+        content = fd.read()
+        # Make sure we're adding the relevant Glean SDK import, once.
+        assert content.count(
+            "import Bar.private.HistogramBase"
+        ) == 1
+
+        # Validate the generated Gecko metric mapper Kotlin operator.
+        # NOTE: Indentation, whitespaces  and text formatting of the block
+        # below are important. Do not change them unless the file format
+        # changes, otherwise validation will fail.
+        expected_operator = """    operator fun get(geckoMetricName: String): HistogramBase? {
```

Comment:
If we wanted to make this more robust to whitespace changes, you could use a regex to remove all whitespace from both the reference and test strings (I think there are other tests that do that).  But not a requirement for this PR, IMHO.

Reply:
Thanks, I tried to do this and it failed due to the nested indentation of the block :o Anyway, given that it's not a requirement, I'll happily skip over this :P